_data/downloads.yml

```diff
@@ -0,0 +1,36 @@
+version: v0.9.2
+
+platforms:
+  - name: Windows
+    assets:
+      - title: Installer
+        description: " (recommended)"
+        url: https://github.com/ActivityWatch/activitywatch/releases/download/v0.9.2/activitywatch-v0.9.2-windows-x86_64-setup.exe
```

Comment:
Since the website is updated every 24h by Travis cron, we could fetch the latest version number (maybe with a few hours lag to ensure that the new packages have been built) and then simply update download.yml with `sed -i 's/v[0-9.]\+/v0.1/g' _data/downloads.yml`.

Reply:
Fixed in https://github.com/ActivityWatch/activitywatch.github.io/pull/12/commits/86099f08594542f6920c6de58025fb162fb4730f